CONTRIBUTING.md

```diff
@@ -52,26 +57,27 @@ To unify the appearance of all commit messages we only accept commit messages us
   > If applied, this commit will ...
 
 #### Correct commits
+
 - ... Add new function to do X #123
 - ... Add test for X #123
 - ... Refactor subsystem X for readability #123
 - ... Update getting started documentation #123
 - ... Remove deprecated methods #123
-    
+
 #### Wrong commits
-- *adding new function to do X for Y* #123
-- *added service X* #123
-- *adds tests for X* #123
-- *fixed bug with Y* #123
-- *changing behavior of Y* #123
-- *more fixes for broken stuff* #123
-- *sweet new API methods* #123
+
+- _adding new function to do X for Y_ #123
+- _added service X_ #123
+- _adds tests for X_ #123
+- _fixed bug with Y_ #123
+- _changing behavior of Y_ #123
+- _more fixes for broken stuff_ #123
+- _sweet new API methods_ #123
```

Comment:
Are the negative examples actually required? I suggest to remove them?

Reply:
I thought it was quite helpful actually.

Reply:
Alright, if you prefer to keep them.